azure-kusto-ingest/azure/kusto/ingest/_resource_manager.py

```diff
@@ -75,6 +77,16 @@ def __init__(self, kusto_client: KustoClient):
         self._authorization_context = None
         self._authorization_context_last_update = None
 
+        self.__set_throttling_settings()
+
+    def __set_throttling_settings(self, num_of_attempts: int = 4, max_seconds_per_retry: float = 30):
```

Comment:
Unit test this? (make sure it rethrows other exceptions and retries with KustoThrottlingError)

Reply:
In my opinion this is a bad practice - you should not write unit tests to verify code that is not yours.
In this case we use external library to implement retry policy, but what if we used external library to parse JSON file? Would I then needed to write unit tests to verify that parsing works as expected?

Reply:
Resolved offline